elements/pfe-band/src/pfe-band.scss

```diff
@@ -0,0 +1,195 @@
+@import "../../pfe-sass/pfe-sass";
+
+$LOCAL: band;
+$aside-sm: 240px;
+$aside-lg: 300px;
+
+:host {
+  display: block;
+
+  // Default spacing values
+  --pfe-band--padding__vertical:                 calc( #{pfe-var(container-spacer)} * 4);
+  --pfe-band--padding__horizontal:               calc( #{pfe-var(container-spacer)} * 1);
+  // Bring them together for ease of use below
+  --pfe-band--padding:                         #{pfe-local(padding__vertical)}  #{pfe-local(padding__horizontal)};
+
+  // Surface colors
+  --pfe-band--backgroundColor:                 #{pfe-color(surface--base)};
+  --pfe-band--backgroundPositionX:             center;
```

Comment:
can we combine these position vars?

Reply:
Yes, just did that in my most recent push.